components/setting_item_min/setting_item_min.tsx

```diff
@@ -44,6 +44,8 @@ interface Props {
      * Update the active section for focusing
      */
     actions: {updateActiveSection: (newActiveSection: string) => {type: string; data: string}};
+
+    focused?: boolean;
```

Comment:
Is this field actually used anywhere? It's not used by this component, and I don't see anywhere that we might be passing it into a child, so if that's the case, we should just remove it.

Reply:
I cannot find any usage either. DesktopNotificationSettings (user_settings/notifications/desktop_notification_settings.tsx) also has a focused field and passes it to SettingItemMin, and this is the only usage from what I can tell. Went ahead and removed it from SettingItemMin, DesktopNotificationSettings, and snapshots for DesktopNotificationSettings. The build passes unit tests and check-types test after this change.